docs/fs.md

```diff
@@ -60,26 +77,45 @@ Assuming you have `focal-server-cloudimg-amd64.raw` and `vmlinux` on your system
     --fs tag=myfs,socket=/tmp/virtiofs,num_queues=1,queue_size=512
 ```
 
-By default, DAX is enabled with a cache window of 8GiB. You can specify a custom size (let's say 4GiB for this example) for the cache by explicitly setting DAX and the cache size:
+By default, DAX is enabled with a cache window of 8GiB. You can specify a custom
+size (let's say 4GiB for this example) for the cache by explicitly setting DAX
+and the cache size:
 
 ```bash
 --fs tag=myfs,socket=/tmp/virtiofs,num_queues=1,queue_size=512,dax=on,cache_size=4G
 
 ```
 
-In case you don't want to use a shared window of cache to pass the shared files content, this means you will have to explicitly disable DAX with `dax=off`. Note that in this case, the `cache_size` parameter will be ignored.
+In case you don't want to use a shared window of cache to pass the shared files
```

Comment:
Why would you not want this?

Reply:
Well first that's to show all the possibilities, but also this prevents the VMM from mapping more memory, as everything goes through the queues in this mode. This can be used for TDX for instance.